spec/unit/views/components/sidebar_section_spec.rb

```diff
@@ -11,36 +11,37 @@
   end
 
   let(:html) do
-    render_arbre_component section: section do
+    result = render_arbre_component section: section do
       sidebar_section(assigns[:section])
     end
+    Capybara.string(result.to_s)
   end
 
   it "should have a title h3" do
-    expect(html.find_by_tag("h3").first.content).to eq "Help Section"
+    expect(html).to have_css '.panel-title', text: 'Help Section'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.